html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<style>
+		html {
+			background-color: #e9eaed;
+			height: 100%;
+			width: 100%;
+		}
+		body {
+			width: 100%;
+			height: 800px;
+			margin: 0;
+		}
+		#Facebook-menu-bar {
+			background-color: #3a5795;
+			color: white;
+			position: relative;
+			top: 0;
+			height: 45px;
+			width: 100%;
+			margin: 0;
+		}
+		#search-bar {
+			position: relative;
+			top: 7px;
+			left: 225px;
+			width: 450px;
+			height: 15px;
```

Comment:
Instead of doing manual placement like this (which is quite rare), it appears that they just have a fixed-width, centered layout. And they just left-alight the search bar.